src/rpcnet.cpp

```diff
@@ -527,6 +528,7 @@ UniValue setban(const UniValue& params, bool fHelp)
             throw JSONRPCError(RPC_MISC_ERROR, "Error: Unban failed");
     }
 
+    uiInterface.BannedListChanged();
```

Comment:
I'm not sure if the locking of a signal emit is done within boost::signal of if this needs explicit locking of `uiInterface`?